src/main/scala/org/broadinstitute/dsde/agora/server/business/PermissionBusiness.scala

```diff
@@ -81,16 +81,21 @@ class PermissionBusiness(permissionsDataSource: PermissionsDataSource)(implicit
   def listEntityPermissions(entities: List[AgoraEntity], requester: String): Future[Seq[EntityAccessControl]] = {
     Future.sequence(entities map {entity =>
       permissionsDataSource.inTransaction { db =>
-        db.aePerms.listOwners(entity)
-      } flatMap { owners =>
-        val entityWithManagers = entity.copy(managers = owners)
+        for {
+          owners <- db.aePerms.listOwners(entity)
+          publicAccess <- db.aePerms.getPermission(entity, "public")
+        } yield {
+          (owners, publicAccess)
+        }
+      } flatMap { case (owners:Seq[String], publicAccess:AgoraPermissions) =>
```

Comment:
What happens if this case fails to match? I *think* it will be a no-op, but not quite sure.